### ISSUE TITLE
marker.dat is not read correctly for European convention

### ISSUE THREAD

**Issue by rafaelhdr** [artoolkit/artoolkit5#3]
I was using multiTest example.
When i execute, it does not work, and the error is

```
Error processing multimarker config file 'Data/multi/marker.dat', marker definition   1: Lines 2 - 4 must be marker transform.
```

It works when i edit the original file *artoolkit/bin/Data/multi/marker.dat*, changing dot to comma.

I think it is a locale problem (i am from Brasil, and the convention here is comma for floats, too)

**Comment by philip-lamb** [comment]
Non-interactive use of (f|s)scanf.*%f to read from datafiles needs to be escaped with setlocale(LC_NUMERIC, "C"); ... setlocale(LC_NUMERIC, "");

The following files are affected:
* lib/SRC/ARWrapper/ARMarker.cpp
* lib/SRC/ARosg/arosg.cpp
* lib/SRC/Eden/glm.c
* lib/SRC/AR2/marker.c
* lib/SRC/AR2/surface.c
* lib/SRC/ARMulti/arMultiReadConfigFile.c
* lib/SRC/ARvrml/arViewerCapi.cpp
* examples/*/VirtualEnvironment(2)?.c
* examples/*/ARMarker(Square|NFT).c

**Comment by philip-lamb** [comment]
setlocale(LC_NUMERIC, "C"); is the default for all executables based on the C runtime, so this would only affect code where a different locale has been set elsewhere. Locale is set per-process.
